File: week1/Emopusta (C# && C++)/C#/ValidAnagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        var sArr = s.ToArray();
        var tArr = t.ToArray();
        Array.Sort(sArr);
        Array.Sort(tArr);
        return (new string(sArr) == new string(tArr));
    }
}

public class Solution {
    public bool IsAnagram(string s, string t) {
        var sArr = s.ToArray();
        var tArr = t.ToArray();
        Array.Sort(sArr);
        Array.Sort(tArr);
        if (sArr.Length != tArr.Length) return false;

        for(int i = 0; i<sArr.Length; i++){
            if (sArr[i] != tArr[i]) return false;
        }
        return true;
    }
}

File: week1/Erenimo3442/valid_anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        letterCount_s = dict()
        letterCount_t = dict()

        for letter in s:
            letterCount_s[letter] = letterCount_s.get(letter, 0) + 1
        for letter in t:
            letterCount_t[letter] = letterCount_t.get(letter, 0) + 1
        
        if letterCount_s == letterCount_t:
            return True
